packages/storage/src/reference.ts

```diff
@@ -201,6 +203,38 @@ export class Reference {
     });
   }
 
+  /**
+   * List items and sub-directories within this directory.
+   *
+   * "/" is treated as a path delimiter. Firebase storage does not support
+   * invalid object path that ends with "/" or contains two consecutive "//".
+   * All invalid objects in GCS will be filtered.
+   *
+   * @param options.maxResults If set, limits the total number of `prefixes`
+   *      and `items` to return.
+   * @param options.pageToken The `nextPageToken` from a previous call to
+   *      list(). If provided, listing is resumed from the previous position.
+   * @return A promise that resolves with the items and sub-directories.
+   *      `prefixes` contains references to sub-directories and `items`
+   *      contains references to objects in this folder. `nextPageToken`
+   *      can be passed as `options.pageToken` to get the rest of results.
+   */
+  list(options?: ListOptions | null): Promise<ListResult> {
```

Comment:
I don't want to make changes twice everytime
TODO(@fredzqm): Once the PR is approved, copy the documentation from `index.d.ts` here.

Reply:
Synced them once. Will do it again after approval.